metabench.cmake

```diff
@@ -70,6 +70,14 @@ function(metabench_add_benchmark target path_to_dir)
         message(FATAL_ERROR "Path specified to add_benchmark (${path_to_dir}) does not contain a chart.json file.")
     endif()
 
+    # Create a dummy executable that will be used to run the benchmark. We'll
+    # run the benchmark through CMake, which allows us to be cross-platform
+    # without additional work.
+    file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/_metabench/${path_to_dir}/measure.cpp "")
+    add_executable(_metabench.${target} EXCLUDE_FROM_ALL ${CMAKE_CURRENT_BINARY_DIR}/_metabench/${path_to_dir}/measure.cpp)
+    set_target_properties(_metabench.${target} PROPERTIES RULE_LAUNCH_COMPILE "${RUBY_EXECUTABLE} -- ${MEASURE_RB_PATH}")
+    set_property(TARGET _metabench.${target} APPEND PROPERTY INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/${path_to_dir}")
```

Comment:
is this equivalent to the less verbose [target_include_directories](https://cmake.org/cmake/help/v3.0/command/target_include_directories.html)?